res/css/structures/_ToastContainer.scss

```diff
@@ -108,12 +108,13 @@ limitations under the License.
         }
 
         .mx_Toast_title {
+            display: flex;
```

Comment:
I don't think we need a flexbox here? What is it fixing?

Reply:
- it aligns the title and the icon (if any)
- it assures the gap between the title and the icon without using margin